docs/uninstall.md

```diff
@@ -0,0 +1,449 @@
+---
+title: "Uninstalling Knative"
+weight: 85
+type: "docs"
+---
+
+To uninstall an Operator-based Knative installation, follow the [Uninstall an Operator-based Knative Installation](#uninstall-an-operator-based-knative-installation) procedure below.
+To uninstall a YAML-based Knative installation, follow the [Uninstall a YAML-based Knative Installation](#uninstall-a-yaml-based-knative-installation) procedure below.
+
+
+## Uninstalling a YAML-based Knative installation
+
+To uninstall a YAML-based Knative installation:
+
+
+### Uninstalling optional Serving extensions
+
+Uninstall any Serving extensions you have installed by following the relevant steps below:
+
+{{< tabs name="serving_extensions" >}}
+
+{{% tab name="HPA autoscaling" %}}
+
+Knative also supports the use of the Kubernetes Horizontal Pod Autoscaler (HPA) for driving
+autoscaling decisions.
+The following command will install the components needed to support HPA-class autoscaling:
```

Comment:
This reads weird, and I think you want to say "remove" rather than "install".

If we delete the namespace at the end, we might not need these, either.

Reply:
Thanks, "Install" was left over from the step I adapted. I'll change that.